.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Before you get started
+- [ ] 🙋‍♀️ Create an issue to discuss what you are doing to do!
+
+---
+
+## Description
+- Closes #issue_number
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 
+
+## Type of change
+Please delete options that are not relevant.
```

Comment:
I find it more convenient with checkboxes (multiple selection allowed). Does markdown support this feature?